src/components/grid-aware/ProgramsBlock/ProgramsBlock.module.css

```diff
@@ -0,0 +1,76 @@
+/* Outter Grid */
+.bleedWrapper {
+  background: var(--color-black);
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-column-start: 2;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding: 100px 0;
+  row-gap: 50px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-bottom: 80px;
+    padding-top: 40px;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaTop {
+  grid-column: 1 / span 10;
+}
+
+.gridAreaBottom {
+  grid-column: 1 / span 12;
+}
+
+
+/* Programs block */
+
+.title {
+  color: var(--color-white);
+  font: var(--font-headline);
+  margin: 0;
+}
+
+
+/* Program card block */
+
+.programsWrapper {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+}
+
+@media (--tablet-and-down) {
+  .programsWrapper {
+    grid-row-gap: 30px;
```

Comment:
According to the designs, it looks like this gap should actually be closer to 40px.

Reply:
The component as a whole has 30px gap

![Screen Shot 2020-11-03 at 4 06 05 PM](https://user-images.githubusercontent.com/30575095/98053981-aefa6700-1dee-11eb-865a-385f2859b290.png)

![Screen Shot 2020-11-03 at 4 06 17 PM](https://user-images.githubusercontent.com/30575095/98054007-be79b000-1dee-11eb-9f9a-99392a6093fd.png)

![Screen Shot 2020-11-03 at 4 05 58 PM](https://user-images.githubusercontent.com/30575095/98054018-c9344500-1dee-11eb-8148-c3505de946db.png)

The textbox in relation to the picture of another card is around 40px although
![Screen Shot 2020-11-03 at 4 06 54 PM](https://user-images.githubusercontent.com/30575095/98054048-d9e4bb00-1dee-11eb-9ae9-c0913d4a6016.png)

![Screen Shot 2020-11-03 at 4 06 49 PM](https://user-images.githubusercontent.com/30575095/98054056-dea96f00-1dee-11eb-8b0d-be5194b171c5.png)

Reply:
I noticed that the spacings between the three cards are inconsistent. The first gap between the first and second card has a spacing of 41px and the second gap between the second and third card has a spacing of 46px. I made a comment on the design to ask @derekfidler for his thoughts on this. In the meantime, I'll change this gap to 40px.

Reply:
No the spacings are good @cw323 , you're looking at the textbox and not the component as a whole